action.yaml

```
@@ -52,7 +52,7 @@ runs:
   steps:
     - name: Check out the repo
       uses: actions/checkout@v3
-      if: ${{ inputs.SKIP_CHECKOUT == 'true' && 'true' || '' }}
+      if: ${{ inputs.SKIP_CHECKOUT == 'false' }}
```

```suggestion
      if: ${{ ! inputs.SKIP_CHECKOUT }}
```
Not sure if that works, can you test?

---

That does not work with `SKIP_CHECKOUT: false`.